jobs/validate-kubeflow-aws/lxc.profile

```diff
@@ -0,0 +1,21 @@
+name: microk8s
```

Comment:
I wonder if this should change. We name it aws during creation. Also this file looks duplicate.

Reply:
Yeah, I just copied the microk8s file since that made the AWS job work. Will update the name, though